docs/SETUP.md

```diff
@@ -131,10 +131,32 @@ jmx:
     keystore_password: ecchronos
     truststore: /path/to/truststore
     truststore_password: ecchronos
-    protocol: TLSv1.2
+    protocol: TLSv1.2,TLSv1.3
     cipher_suites:
 ```
 
+CQL also supports certificates in PEM format.
+
+```
+cql:
+  credentials:
+    enabled: true
+    username: cassandra
+    password: cassandra
+  tls:
+    enabled: false
+    certificate: /path/to/certificate
+    certificate_key: /path/to/certificate_key
+    certificate_authorities: /path/to/certificate_authorities
+    protocol: TLSv1.2,TLSv1.3
+    cipher_suites:
+    require_endpoint_verification: false
+```
+> **Note**
+>
+> In case certificate stores and PEM certificates are declared in `conf/security.yml` for CQL,
```

Comment:
We say that if both certificates and keystore/truststore is specified, the pem file takes precedence. How does it work with springboot? Is the behavior the same? If it is, we should put a note in the application.yaml file. We should aim the have the same behavior in both places.